doc/docs/development/run-unit-tests.md

```diff
@@ -1,11 +1,12 @@
 # cargo-nextest
```

Comment:
This section is somehow falling out of line compared with the rest of the documentation. Maybe we can do some improvements here too:
```suggestion
# Unit tests with cargo-nextest
```

Reply:
I hope, the tooling does not annoy you :). With the VS.Code plugin, I find it nice and supportive. Without (and only running in the CI), it is as annoying as Checkstyle ^^

Reply:
🤣 No, the changes are great. I just have the habit to directly fix things I see. We could have moved it out into another PR, but it seems like too much work for a single line 😉